.github/workflows/compute.yml

```diff
@@ -0,0 +1,52 @@
+name: "@ubiquity-os-marketplace/command-ask"
+
+on:
+  workflow_dispatch:
+    inputs:
+      stateId:
+        description: "State Id"
+      eventName:
+        description: "Event Name"
+      eventPayload:
+        description: "Event Payload"
+      settings:
+        description: "Settings"
+      authToken:
+        description: "Auth Token"
+      ref:
+        description: "Ref"
+      signature:
+        description: "Signature sent from the Kernel"
+
+jobs:
+  compute:
+    name: "LLM Reply"
+    runs-on: ubuntu-latest
+    permissions: write-all
+    env:
+      SUPABASE_URL: ${{ secrets.SUPABASE_URL }}
+      SUPABASE_KEY: ${{ secrets.SUPABASE_KEY }}
+      VOYAGEAI_API_KEY: ${{secrets.VOYAGEAI_API_KEY}}
+      OPENAI_API_KEY: ${{secrets.OPENAI_API_KEY}}
+      UBIQUITY_OS_APP_NAME: ${{secrets.UBIQUITY_OS_APP_NAME}}
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20.10.0"
+
+      - name: install dependencies
+        run: yarn
+
+      - name: execute directive
+        run: npx tsx ./src/main.ts
```

Comment:
```suggestion
        run: yarn tsx ./src/main.ts
```

Reply:
If you are using both indeed maybe not. I think you can include that file within `knip` configuration instead, in the `entry` section
https://knip.dev/overview/configuration#defaults